### PR TITLE
Update resources

### PR DESCRIPTION
**Pull request by dholbach**
Fixes: #457

Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Comment by dholbach** [comment]
Still needs a nicer preview for youtube playlist links.

**Comment by dholbach** [comment]
:heavy_check_mark:

**Comment by dholbach** [comment]
@staceypotter Is this what you had in mind? https://deploy-preview-459--fluxcd.netlify.app/

**Comment by staceypotter** [comment]
Looks good, thanks Daniel!  I see what you mean about the "nicer preview" and agree. I also wonder about limiting character count in the descriptions - does 200 (with spaces) work do you think?

**Comment by dholbach** [comment]
I'll see what I can do.

**Comment by dholbach** [comment]
Better now?

File: content/en/_index.html [hunk 1]
```diff
@@ -183,37 +183,39 @@ <h2 class="section-label">Resources</h2>
...
+  Leigh Capili gives us a black belt, under-the-covers look at how Flux's kustomize-controller tracks the child objects for a Kustomization in the Kubernetes API.
   {{% /blocks/resource %}}
 
   {{% blocks/resource
-    youtube="R6OeIgb7lUI"
-    title="Flux v2 overview, demo, & review with Viktor Farcic" %}}
-  Stefan introduces Flux v2, explains why it was reshaped into a composable continuous delivery solution that goes beyond Git sync to accommodate multi-tenancy, infrastructure dependencies and cluster-api fleet management.
+    youtube="laMwuG8r7Tw"
+    title="Handling Dependencies with Flux with Jason Morgan" %}}
+  Kubernetes provides a simple way to apply configuration against a cluster but sometimes the order of operations matters.
```
Comment (staceypotter) [inline content/en/_index.html:213]: Jason walks through how to use Flux’s dependency manager to ensure that your apps get added to your mesh every time.

**Comment by dholbach** [comment]
This is ready for review now.

**Review by hiddeco — APPROVED** [review]

**Pull request merged**